tests/flux-automation.sh

```diff
@@ -19,6 +19,12 @@ exclusions=(
   k8s/aat/common/idam/idam-api.yaml
   k8s/aat/common/idam/idam-web-admin.yaml
   k8s/aat/common/idam/idam-web-public.yaml
+  k8s/prod/common/dg/dg-docassembly.yaml
```

Comment:
do we need a comment explaining why and how long its valid for?

Reply:
I am not sure I want to merge it tbh, maybe better leave it broken as a reminder